### PR TITLE
added link to routing information

### PR DESCRIPTION
**Pull request by dampee**
Forgot that the backend needs different routing, and this pages wasn't mentioning it.

### CODE REVIEW

File: Reference/Routing/WebApi/authorization.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ _This section will describe how to secure your Umbraco Api controllers based on
 
 Probably the easiest way to ensure your controller is secured for only back office users is to inherit from `Umbraco.Web.WebApi.UmbracoAuthorizedApiController`. This is essentially the same as applying the `UmbracoAuthorizeAttribute` to your controller (see below). We also expose the Umbraco user property `User UmbracoUser {get;}` on this base controller as well.
 
+The `UmbracoAuthorizedApiController` ìs automatically routed.  Check out the [routing documentation](../Authorized/index.md) for more information on this topic.
```
Comment (hemraker) [inline Reference/Routing/WebApi/authorization.md]: I do believe the ì in is should be an i :smiley:

**Comment by hemraker** [comment]
Nice addition! Left a comment for a tiny change.

**Comment by dampee** [comment]
:+1:   You must have an hawk eyes, I've send an update

**Comment by hemraker** [comment]
Seems to work best when reviewing :wink: Thanks a bunch!!

**Pull request merged**